### PR TITLE
feat: Cart and Wishlist pages added

### PR DESCRIPTION
**Pull request by Rishabh-078**
**Cart Page:** 

cart

**Wishlist Page:**

wishlist

### CODE REVIEW

**Review by malaykhakhar — APPROVED** [review]
The code looks good and clean. You can merge it

File: cart.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+@import "./style/footer.css";
+@import "./style/cards.css";
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+:root {
```
Comment (prankurpandeyy) [inline cart.css:11]: the values here are mixed ,like you should either use color name or color code not both

File: cart.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    --primary-color: #a23df1;
+    --secondary-color: white;
+    --tertiary-color: grey;
+    --primary-color-b: #D2A0F8;
+    --quaternary-color: rgb(8, 100, 8);
+    --shadow-color: rgb(0 0 0 / 40%);
+}
+
+button{
```
Comment (prankurpandeyy) [inline cart.css]: please avoid using direct global tags it will cause issues later

**Review by prankurpandeyy — CHANGES_REQUESTED** [review]

File: cart.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                </div>
+            </div>
+            <div class="horizontal-card">
+                <img class="img-card-horizontal" src="Assets/Outdoor/gladiolus-random-color.jpg" alt="Gladiolus flower" loading="lazy"/>
+                <div class="card-horizontal-desc">
+                    <p>Gladiolus flower bulbs (Random color)</p>
+                    <div class="horizontal-price"><span class="stroked-price"><i
+                                class="fas ruppee-stroke fa-rupee-sign"></i> 200</span>
+                        <span class="actual-price bold"><i class="fas ruppee fa-rupee-sign"></i> 189</span></div>
+                    <div class="disc-percent">10%</div>
```
Comment (swstica) [inline cart.html:62]: IMO, you can write like "10% off". Just "10%" looks confusing.

**Review by swstica — APPROVED** [review]
Rest all looks good:))

**Review by prankurpandeyy — APPROVED** [review]

File: cart.html (hunk 3 above)
Comment (Rishabh-078) [reply cart.html:62]: Yeah, will change it

**Pull request merged**